.github/dependabot.yaml

```diff
@@ -0,0 +1,49 @@
+---
+updates:
+  - directory: "/"
+    package-ecosystem: "github-actions"
+    schedule:
+      interval: "weekly"
+  - directory: "/"
+    package-ecosystem: "npm"
+    schedule:
+      interval: "weekly"
+  - directory: "/src"
+    package-ecosystem: "npm"
+    schedule:
+      interval: "weekly"
+  - directory: "/src/forwarder"
+    package-ecosystem: "npm"
+    schedule:
+      interval: "weekly"
+  - directory: "/src/poller"
+    package-ecosystem: "npm"
+    schedule:
+      interval: "weekly"
+  - directory: "/src/poller-core"
+    package-ecosystem: "npm"
+    schedule:
+      interval: "weekly"
+  - directory: "/src/scaler"
+    package-ecosystem: "npm"
+    schedule:
+      interval: "weekly"
+  - directory: "/src/scaler-core"
+    package-ecosystem: "npm"
+    schedule:
+      interval: "weekly"
+
+  - directory: "/poller"
+    package-ecosystem: "docker"
+    schedule:
+      interval: "weekly"
+  - directory: "/scaler"
+    package-ecosystem: "docker"
+    schedule:
+      interval: "weekly"
+  - directory: "/forwarder"
+    package-ecosystem: "docker"
+    schedule:
+      interval: "weekly"
+version: 2
```

Comment:
Do we also want /src in this list for docker?

Reply:
Fixed. 

Also removed the additional package.json and package-lock.json files